exla/mix.exs

```diff
@@ -167,4 +173,16 @@ defmodule EXLA.MixProject do
       :filename.basedir(:user_cache, "xla")
     end
   end
+
+  # Returns `path` relative to the `from` directory.
+  defp relative_to(path, from) do
```

Comment:
Note that this doesn't consider symlinks, but we don't have a top-level priv in exla that mix would symlink to, so this should be fine (?)